Displays/PIXIE.txt

```diff
@@ -0,0 +1,117 @@
+PIXIE
+
+
+DCPU-16 Hardware Info:
+    Name: PIXIE - Compatible Video Generator
+    ID: 0x774df615, version: 0x1802
+    Manufacturer: 0x00000000 (Manufacturer not yet found)
+
+
+Description:
+    The PIXIE is a 128x96 pixel color display fully compatible with the
+    famous Nya Elektriska's LEM1802. PIXIE is a perfect way to upgrade
+    your DCPU and start enjoying the latest games and graphics.
+
+    PIXIE has the same interrupt behaviour as LEM1802, but it also supports
+    a 128x96 resolution graphics mode with 2-16 colors. Graphics mode is
+    enabled with a new interrupt that specifies the number of colors.
+    If using less then 16 color palette, the lowest colors in LEM1802 text
+    mode palette are used.
+
+
+Interrupt behavior:
+    When a HWI is received by the PIXIE, it reads the A register and does one
+    of the following actions:
+
+    Interrupts 0 to 5 are compatible with LEM1802.
+
+    0: MEM_MAP_SCREEN
+       Reads the B register, and maps the video ram to DCPU-16 ram starting
+       at address B. See below for a description of video ram.
+       If B is 0, the screen is disconnected.
+    1: MEM_MAP_FONT
+       Reads the B register, and maps the font ram to DCPU-16 ram starting
+       at address B. See below for a description of font ram.
+       If B is 0, the default font is used instead.
+    2: MEM_MAP_PALETTE
+       Reads the B register, and maps the palette ram to DCPU-16 ram starting
+       at address B. See below for a description of palette ram.
+       If B is 0, the default palette is used instead.
+    3: SET_BORDER_COLOR
+       Reads the B register, and sets the border color to palette index B&0xF
+    4: MEM_DUMP_FONT
+       Reads the B register, and writes the default font data to DCPU-16 ram
+       starting at address B.
+       Halts the DCPU-16 for 256 cycles
+    5: MEM_DUMP_PALETTE
+       Reads the B register, and writes the default palette data to DCPU-16
+       ram starting at address B.       
+       Halts the DCPU-16 for 16 cycles
+
+    16: SET_VIDEO_MODE
+       Reads the B register, and changes to specified video mode:
+
+          0: Text mode (LEM1802 standard)
+          1: Graphics mode, 2 colors
+          2: Graphics mode, 4 colors
+          3: Graphics mode, 8 colors
+          4: Graphics mode, 16 colors
+
+       Graphics mode will use the same video_ram starting address as text mode,
+       but video_ram size and layout are different. Graphics mode will also use
+       the same palette as text mode, but only the colors 0-n (depending on the
+       color depth) are used. Fontram is not relevant in graphics mode.
```

Comment:
font ram and video ram are referred to via "font ram" and "video ram", could we drop the underscores, and add a space to Fontram?